src/app/top-menu-bar/top-menu-bar.component.scss

```diff
@@ -0,0 +1,14 @@
+.mat-toolbar.mat-primary {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
+.app-title {
+  padding-left: 20px;
+}
+
+.main-toolbar {
+  z-index: 5 !important;
+  will-change: transform;
```

Comment:
I'd never heard of this one. Nifty!

Reply:
Turns out I was using that completely wrong. Don't know where I got the idea from.

Reply:
Ah, roger that, then! ☺️